docs/GPU-Windows.rst

```diff
@@ -397,6 +411,7 @@ Now that you compiled LightGBM, you try it... and you always see a segmentation
 .. image:: ./_static/images/screenshot-segmentation-fault.png
    :align: center
    :target: ./_static/images/screenshot-segmentation-fault.png
+   :alt: A screenshot of the command prompt where a segmentation fault has occured while using Light B G M
```

Comment:
```suggestion
   :alt: A screenshot of the command prompt where a segmentation fault has occurred while using Light G B M
```

I make this mistake a lot in this project haha

Reply:
Dang T_T